scenarios/CreateAKSDeployment/README.md

```diff
@@ -20,18 +20,11 @@ The first step in this tutorial is to define environment variables.
 
 ```bash
 export RANDOM_ID="$(openssl rand -hex 3)"
-export NETWORK_PREFIX="$(($RANDOM % 254 + 1))"
 export SSL_EMAIL_ADDRESS="$(az account show --query user.name --output tsv)"
-export MY_RESOURCE_GROUP_NAME="myAKSResourceGroup$RANDOM_ID"
+export MY_RESOURCE_GROUP_NAME="myResourceGroup$RANDOM_ID"
```

Comment:
Can we change this back to myAKSResourceGroup$RANDOM_ID?  It can get confusing with many exec docs if they all have the same prefix name.

Reply:
done